src/type-layout.md

```diff
@@ -264,13 +264,23 @@ For all other enumerations, the layout is unspecified.
 
 Likewise, combining two primitive representations together is unspecified.
 
-### The `packed` Representation
+### `align(x)` Representation
+
+The `align(x)` representation can be used on `struct`s and `union`s to raise the
```

Comment:
Remove the `(x)` from the name. Instead, describe how it takes the alignment as a parameter in the paragraph describing how it works.